.github/workflows/ci.yml

```diff
@@ -1,31 +1,125 @@
+# Copyright 2020 The Cockroach Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
 name: CI
 on: [push, pull_request]
 
 jobs:
   test:
+    # Execute our own tests
+    # This one will spawn 6 jobs
+    name: Main - CRDB ${{ matrix.cockroachdb-version }}, Sequelize v${{ matrix.sequelize-version }}
     strategy:
       matrix:
         sequelize-version: [5, 6]
+        cockroachdb-version: [latest, v20.2.2, v19.2.11]
```

Comment:
can we also test with v20.1?

also, in case you would like, we have the following tags to point to the latest version of each major release: latest-v19.2, latest-v20.1, latest-v20.2. on the other hand, i see an advantage to using exact release tags so that the tests are more stable (and we will update the tags as needed)

Reply:
Hmm, unfortunately GitHub actions is limited to 256 jobs per workflow, so just adding another version would go over that limit. I think it might be possible to work around this limit using several workflows, but I'd have to test. I never had to start so many jobs in a GitHub action before.

Another option would be to see this just as a temporary thing; this "file-splitting" was done in order to allow us to see more tests running, since unfortunately there are several tests that are not only failing but also crashing the entire test execution. Once these problems start to be fixed, the amount of tests that crash the execution will get smaller, and ideally at some point will be brought down to zero, and then we will be able to run all Sequelize tests at once.

I just thought of another option as well: assuming that the nature of these crashes does not change across different CockroachDB versions, what we could do is take note of each test that is crashing the suite, and mark them to be skipped. And then redo this process until all tests that crash the suite are skipped. This way we could run all the other tests in a single take. What do you think?

Reply:
As a quick way to address your specific concern though, I could just do:

```diff
- [latest, v20.2.2, v19.2.11]
+ [v20.2.2, v20.2.1, v19.2.11]
```

Since if I'm not mistaken `latest` is the same as `v20.2.2` right now (altough of course this will change with time)

Reply:
I agree -- let's do the file-splitting approach as a temporary thing until we can figure out which tests are stable (or fix them within the DB or test suite as needed).

For now though, I was asking to test with `v20.1`. so let's make the list:

```
[v20.2.2, v20.1.9]`
```

we don't need 19.2 since there are no more scheduled releases for it.

Reply:
@rafiss Oops, actually I made a mistake. The line we're discussing is actually for the tests from this repo - they run fine in a single job per version combination (the `test` block from `ci.yml`). For what I said above, I was thinking it was for the `sequelize-postgres-integration-tests` block, which is the one that performs the file-splitting. Sorry for mixing it up.

So, correcting myself now:

* It is easy to apply your suggestion, we can even do `[latest, v20.2.2, v20.1.9, v19.2.11]`
* Also, in fact my PR already does the file-splitting only for `v20.2.2`, but does it for both Sequelize v5 and v6. We can also do for both `v20.2.2` and `v20.1.9` if you want, but then it would need to be only with Sequelize v6 (or v5), otherwise it would put us over the limit...

Reply:
@rafiss I will try to look into a way of increasing (or working around) this limit so that we can test with CockroachDB `v20.2.2` and `v20.1.9` and also Sequelize v5 and v6. In the meantime, please tell me what you'd prefer:

* Test both CockroachDB `v20.2.2` and `v20.1.9`, but only with one Sequelize version (probably v6, I would guess)
* Test only one version of CockroachDB (which one?), with both Sequelize v6 and v5

Let me know and I'll update this PR. I'll post here if I figure out a way to work around this limitation.

Reply:
@rafiss Great, this is what is already in place (except that it is using 20.2.2, but there is an open PR (#54) that updates it to 20.2.4. By the way, if you always want to test with the latest version, you can use the `latest` keyword instead of providing a fixed version number, and the latest version available on Docker will be taken at the moment the GitHub Action starts.